### PR TITLE
NewModes - Add POOL OF VEHICLEs, Also fix up merge of Readme & netex_publication files

### PR DESCRIPTION
**Pull request by nick-knowles**
Adds new PoolOfVehicles element as comments last Friday.

Also resolves issues  arising form merge by SK, CD, NK on 01/09/201
   (1) Remove duplication in netex_publication file
   (2) Add lost changes to README and clean up. 
   (3) Remove ambiguity introduced into accessRightParameter file that broke the XML

Has been revalidated against examples.

### CODE REVIEW

**Comment by nick-knowles** [comment]
This will be urgent to apply as the new-modes branch is currently broken.

File: xsd/netex_part_5/part5_frames/netex_nm_mobilityServiceFrame_version.xsd [hunk 1]
```diff
@@ -47,7 +49,7 @@
 					<Requires>http://www.netex.org.uk/schemas/1.0/ifopt/netex_xxxxx.xsd</Requires>
 				</Relation>
 				<Rights>Unclassified
- <Copyright>CEN, Crown Copyright 2009-2017</Copyright>
+                    <Copyright>CEN, Crown Copyright 2020-2021</Copyright>
```
Comment (skinkie) [inline xsd/netex_part_5/part5_frames/netex_nm_mobilityServiceFrame_version.xsd:52]: Alignment.

File: xsd/netex_part_5/part5_rc/netex_nm_mobilityService_version.xsd [hunk 2]
```diff
@@ -507,6 +516,12 @@ Rail transport, Roads and Road transport
 		</xsd:annotation>
 		<xsd:sequence>
 			<xsd:element ref="VehicleSharingRef"/>
+			<xsd:element name="SharingPolicyUrl" type="xsd:anyURI" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>URL for info on Sharing policy.
+</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_5/part5_rc/netex_nm_mobilityService_version.xsd:522]: alignment

**Comment by nick-knowles** [comment]
Stefan I suggest that the only sensible way to deal with README & netex_publication is just to replace - I started with the versions we had this morning so they should include all changes from master.

File: xsd/netex_part_5/part5_nd/netex_nm_mobilityServiceConstraintZone_support.xsd [hunk 3]
```diff
@@ -22,6 +22,10 @@
 				</Date>
 				<Date>
 					<Modified>2021-05-03</Modified>Add zone rule applicability values
+				</Date>
+				<Date>
+					<Modified>2021-09-01</Modified>NewModes Comments: Add __mustpickupanddropoffInSameZone__ value
+					    Add PoolOfVehicles
 				</Date>>
```
Comment (Aurige) [inline xsd/netex_part_5/part5_nd/netex_nm_mobilityServiceConstraintZone_support.xsd]: This tag looks weird

**Comment by skinkie** [comment]
@nick-knowles that is not a sensible way. Because that will prevent merging. So if you want to start over: start with the current xsd from master and include the new modes differences. Or (obviously better): fix the changes that break the current file.

**Review by Aurige — COMMENTED** [review]
> 
> 
> @nick-knowles that is not a sensible way. Because that will prevent merging. So if you want to start over: start with the current xsd from master and include the new modes differences. Or (obviously better): fix the changes that break the current file.

yes, we need to avoid to redo what we did this morning !

**Review by Aurige — CHANGES_REQUESTED** [review]
Ok for me but we need to solve the README issue before merging

File: xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd [hunk 4]
```diff
@@ -412,12 +416,6 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>ORGANISATION validity parameters for ACCESS RIGHT PARAMETER ASSIGNMENT.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:sequence>
-			<xsd:element name="VehicleModes" type="VehicleModeListOfEnumerations" minOccurs="0">
```
Comment (skinkie) [inline xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd:415]: Why in this one removed?

Comment (Aurige) [reply xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd:415]: This file is for ACCESS RIGHT for FARES, and the this  OrganisationValidityParametersGroup is to connect them to ORGANISATION, not to MODEs, there is a ModeValidityParametersGroup for this. So this looks Ok (was here by mistake I guess)

**Review by Aurige — APPROVED** [review]
I think that it should be Ok now
can you double check @skinkie ?

**Pull request merged**